_topic_maps/_topic_map.yml

```diff
@@ -93,6 +93,8 @@ Distros: openshift-gitops
 Topics:
 - Name: Configuring an OpenShift cluster by deploying an application with cluster configurations
   File: configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations
+- Name: Managing the application set resources in non-control plane namespaces
```

Comment:
```suggestion
- Name: Managing the applicationset resources in non-control plane namespaces
```

Reply:
@varshab1210 and @svghadi As per this guideline - https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting, we have to keep it separate and we are retaining it as is here.